### PR TITLE
credit card authorization

### PR DESCRIPTION
**Pull request by michal-at-travisci**
billing overview update

### CODE REVIEW

**Review by stan-travis — APPROVED** [review]
Looks good to me.

**Review by stan-travis — APPROVED** [review]

File: user/billing-faq.md [hunk 1]
```diff
@@ -9,8 +9,7 @@ permalink: /user/billing-faq/
 
 ## How can I get on an annual plan? 
 
-The annual based plans are available by contacting the Travis CI support team. 
-
+You can select one fo available annual plans via [Your Account->Settings->Plan](https://app.travis-ci.com/account/plan) page. The customised annual based plans are available by contacting the Travis CI support team.
```
Comment (stan-travis) [inline user/billing-faq.md:12]: It should be "You can select one of available annual plans"

File: user/billing-faq.md [hunk 2]
```diff
@@ -28,13 +27,15 @@ By default Travis CI system provides the possibility to trigger a build to all m
 If the team member has not triggered the build during the billing period Travis CI will not charge you for that user.
 
 
-To check how much active users you got during the last billing cycle please contact the support.
-Travis CI is working on the user management functionality where you will be able:
+To check how much active users you got during the last billing cycle you may generate a report for selected timeriod on your [Plan Usage](https://app.travis-ci.com/account/plan/usage) page or contact the support.
```
Comment (stan-travis) [inline user/billing-faq.md:30]: Should be "time period"

File: user/billing-faq.md [hunk 3]
```diff
@@ -50,23 +51,22 @@ You may have been assigned an amount of OSS credits to run builds on public repo
 
 ## How do I recharge my credits balance?
 
-You can buy additional build credits anytime you need them by clicking on your profile icon in the right upper corner of the screen =>Settings, navigate to the Plan page and  press the ‘Buy add-ons’ button.
-Please be advised that it is not possible to buy additional credits on Free Plan. 
+You can buy additional build credits anytime you need them by clicking on your profile icon in the right upper corner of the screen =>Settings, navigate to the Plan page and  press the ‘Buy add-ons’ button. You may also enable the [auto-refill](user/billing-autorefill/) feature.
+Please be advised that it is not possible to buy additional credits on free Plan.
```
Comment (stan-travis) [inline user/billing-faq.md:55]: Shouldn't we call it Free Trial Plan?

File: user/billing-faq.md [hunk 4]
```diff
@@ -50,23 +51,22 @@ You may have been assigned an amount of OSS credits to run builds on public repo
...
-You can buy additional build credits anytime you need them by clicking on your profile icon in the right upper corner of the screen =>Settings, navigate to the Plan page and  press the ‘Buy add-ons’ button.
-Please be advised that it is not possible to buy additional credits on Free Plan. 
+You can buy additional build credits anytime you need them by clicking on your profile icon in the right upper corner of the screen =>Settings, navigate to the Plan page and  press the ‘Buy add-ons’ button. You may also enable the [auto-refill](user/billing-autorefill/) feature.
+Please be advised that it is not possible to buy additional credits on free Plan. 
 
 
 ## Do credits expire?
 
-No, the credits you purchased do not expire. 
+Credits which you purchase or obtain as a part of the free Plan do have an expiry date. Please check your [Plan Usage]((https://app.travis-ci.com/account/plan/usage) page.
```
Comment (stan-travis) [inline user/billing-faq.md:60]: Shouldn't we call it Free Trial Plan?

**Review by stan-travis — APPROVED** [review]
Approved

**Review by travis-architect — APPROVED** [review]

**Pull request merged**